docs/conf.py

```diff
@@ -133,14 +128,14 @@ def get_tutorial_images():
 source_suffix = '.rst'
 
 # The encoding of source files.
-#source_encoding = 'utf-8-sig'
+# source_encoding = 'utf-8-sig'
 
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
 project = 'MSS - Mission Support System'
-copyright = \
+project_copyright = \
```

Comment:
this is a var which is used by the template.

and is used on bottom of the rtfd page

![image](https://github.com/Open-MSS/MSS/assets/278728/dad26781-122c-4d49-b7e4-616f9afed98f)

Reply:
https://github.com/readthedocs/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/footer.html#L21

Reply:
`project_copyright` was specifically added as an alias for `copyright` to fix this issue: https://github.com/sphinx-doc/sphinx/issues/8132

Reply:
the template in ReadTheDocs still uses that oldvalue. So you can add a ToDo there for renaming once it is fixed or send a PR to the template too. And when upstream it is changed we can use it.

Reply:
It is an alias, `copyright` uses the value of `project_copyright` as a default, so it is set to the same value: https://github.com/tk0miya/sphinx/blob/c5a9d04d453f457b087a03037ce49f88a4e439cb/sphinx/config.py#L101-L102. The template can continue to use `copyright`, that is not an issue.

I've also just validated that it does indeed work.